src/applications/Job.java

```diff
@@ -30,6 +31,17 @@ public int removeNextTask() {
         return theTime;
     }
 
+
+    /**
+     * This method extracts the Machine number from the job
+     * @return
+     */
+    public int getMachineNumber(){
```

Comment:
This is a nice change that definitely makes things more clear in line 36 of MachineShopSimulator.